src/libaktualizr/package_manager/dockerappmanager.cc

```diff
@@ -27,6 +27,8 @@ DockerAppManagerConfig::DockerAppManagerConfig(const PackageConfig &pconfig) {
   }
 
   if (raw.count("docker_prune") == 1) {
-    docker_compose_bin = raw.at("docker_prune");
+    std::string val = raw.at("docker_prune");
+    boost::algorithm::to_lower(val);
+    docker_prune = val == "0" || val == "false";
```

Comment:
This logic seems backwards to me... is it missing a `!`?

Reply:
wow - the fact I've messed this up 2 times now seemed like the CI system was crying for me to help. This is fixed and I've sprinkled in some unit testing to help.